uint16 mapID
uint16 mapType
char[64] mapName
uint32 mapSize